src/main/scala/com/raquo/airstream/combine/generated/CombinableSignal.scala

```diff
@@ -0,0 +1,253 @@
+package com.raquo.airstream.combine.generated
+
+import app.tulz.tuplez.Composition
+import com.raquo.airstream.core.Signal
+
+// #Warning do not edit this file directly, it is generated by GenerateCombinableSignal.scala
+
+// These combine / combineWith / withCurrentValueOf / sample methods are implicitly available on all signals
+// For combine / combineWith methods on the Signal companion object, see StaticSignalCombineOps.scala
+
+class CombinableSignal[A](val signal: Signal[A]) extends AnyVal {
+
+  def combineWith[T1](
```

Comment:
Just a boring lil' naming suggestions:

Have you considered `zip` for this one? And `zipWith` for `combineWithFn`? 
I like the terseness/analogousness to List's zip. But I can live either way! :D

This update looks awesome btw!

Reply:
I'd love to have `zip`, too. 

But it appears that `zip` has different semantics from what we have here (see Nikita's comment [here](https://github.com/raquo/Airstream/pull/51#issuecomment-751306016)).

(I, personally, could live with this inconsistency vs RxJS (at least for Signals) :) )

Reply:
As explained in the linked comment, zip has legitimately different semantics from combine, it's not just a feature of ReactiveX, Monix and all other streaming libraries I know behave the same way. If we had an operator called zip that did a different thing than what's standard, many people would consider it a bug.

On lists, "zip" seems like the best match to what we're doing only because combineLatest can't exist on a list, since there is no concept of time in a list.